### ISSUE TITLE
ion-img not update after fully rendered

### ISSUE THREAD

**Issue by jccf091** [driftyco/ionic#6285]
#### Short description of the problem:
ion-img cannot load image due to the element is not visiable at first 

#### What behavior are you expecting?
070133848409773bc104684aefb33c2cb6e9dbe5
the image should be load when nativeElement change to visible
When I set the src of the ion-img, it will call _update fn. But the this.isVisible fn will always return false because the bottom of the "bounds" is 0 at that moment. 

After the page was fully rendered, I inspected the nativeElement of 'ion-img'. I found that the bounds.bottom is 44.

I expected the update fn will be called  when nativeElement change.

```
<ion-img
  [src]="'https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSfPcEyKKYBIHuA7oBssjIm7xm0A_dSE17oE3H5olrNO_xNUdkX0w'"
  [height]="100"
  [width]="100">
</ion-img>
```
#### 2.0.0-beta.6

**Comment by jccf091** [comment]
We should use Renderer to listen nativeElement's changes.

**Comment by ghenry22** [comment]
I also logged this under #6257

**Comment by jgw96** [comment]
Hello all! Thanks for opening an issue with us! Is this happening when using ion-img in a standard list or with virtualscroll?

**Comment by jccf091** [comment]
@jgw96 
I used ion-img directly under ion-content open tag and without virtualscroll

**Comment by jgw96** [comment]
OK, thanks for the follow up ! So ion-img is only meant to be used with VirtualScroll right now, so using it outside of virtualscroll is not going to work correctly for now. We recommend using a regular IMG element and only using ion-img with VirtualScroll. Thanks for using Ionic! I am going to close this now, but feel free to comment!

**Comment by nsuyog** [comment]
ion-img not rendered 
If i used *ng-for on parent element then ion-img not rendered. I didn't used any virtualscroll. 
Is this issue fixed on Ionic Version: 3.2.1

![image](https://user-images.githubusercontent.com/23333399/27631867-a563ac98-5c17-11e7-9b99-54fca4baf209.png)
